cmd/snap/cmd_man.go

```diff
@@ -0,0 +1,44 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2014-2015 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"os"
+
+	"github.com/ubuntu-core/snappy/i18n"
+)
+
+type cmdMan struct{}
+
```

Comment:
I'd love not to nitpick on this one but perhaps we could hide this command?

``commad.Hidden = true`` is all that is needed (we could tweak addCommand() to return the added instance in case it doesn't already do this). 

Then the long/short help would not matter much